config.py

```diff
@@ -568,7 +568,7 @@
 # AUTOMATIC REJECTION OF ARTIFACTS
 # --------------------------------
 
-reject: Optional[dict] = {'grad': 4000e-13, 'mag': 4e-12, 'eeg': 150e-6}
+reject: Union[dict, Literal['auto']] = {'grad': 4000e-13, 'mag': 4e-12, 'eeg': 150e-6}
```

Comment:
I'm actually thinking we should default this to `None` or `'auto'` now. @agramfort WDYT?